docs/conf.py

```diff
@@ -96,7 +96,7 @@
     'github_user': 'jupyter',
     'github_repo': 'docker-stacks',
     'github_type': 'star',
-    'logo': 'jupyter-logo.png',
+    'logo': 'jupyter-logo.svg',
```

Comment:
This one looks smaller, but at the same time much better on retina screens.
Took the file from here: https://upload.wikimedia.org/wikipedia/commons/3/38/Jupyter_logo.svg